src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -283,68 +404,52 @@ export default class CanvasController {
                 round(this.canvas.size.width / 2 + this.canvas.panX),
                 round(this.canvas.size.height / 2 + this.canvas.panY)
             );
-            // TODO: this is incorrect usage of a ternary. ternaries should be used for a ssignment not function calls.
-            // draw airspace border
-            AirportController.airport_get().airspace
-                ? this.canvas_draw_airspace_border(cc)
-                : this.canvas_draw_ctr(cc);
+
+            if (AirportController.airport_get().airspace) {
+                this.canvas_draw_airspace_border(cc);
+            } else {
+                this.canvas_draw_ctr(cc);
```

Comment:
FWIW, a `airspace` property is now guaranteed by design. This is left over from when we could have circular airspaces around the center of the airport, and only define the radius. Now we must have GPS coordinates for the airspace border. So this conditional and the `canvas_draw_ctr()` could definitely be removed if you're so inclined.

Reply:
excellent 🎉